cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -216,6 +216,14 @@ message SearchSeismicStoresRequest {
     CoverageSpec coverage = 7; // If specified, include coverage
 }
 
+message InspectIngestionRequest {
+    oneof identifier {
+        int64 seismic_store_id = 1;
+        int64 file_id = 2;
+    }
+    bool allow_unpublished = 3;
```

Comment:
Is there much point to having a flag for this, or will they just always set it to true for this purpose?

Reply:
Not really. I just wanted to be really explicit to make sure people know what they are doing. But maybe not worth it here.